### ISSUE TITLE
Suddenly on 4-8-2018 can't access FB messages! New version doesn't help!

### ISSUE THREAD

**Issue by JosieDraus** [indywidualny/FaceSlim#329]
On Sunday, 4-8-2018, I lost access to FB messenger!. Before that FaceSlim did both FB & Messenger, but now it doesn't! Help!

**Comment by schmatzler** [comment]
Change the user agent to:

    Mozilla/5.0 (BB10; Kbd) AppleWebKit/537.35+ (KHTML, like Gecko) Version/10.3.3.3057 Mobile Safari/537.35+

and it will work.

**Comment by schmatzler** [comment]
Switch to touch.facebook.com in the settings and it will look better.

**Comment by ZonaRMR** [comment]
No problem for me, Facebook messenger is working fine.

**Comment by michaelsteffen1982** [comment]
@schmatzler : I am also using a Q5 and the latest OS - but it does not work.
If I leave the referer empty+activate 'touch' in the options, FB looks as ever - but I can´t access messages.
If I use the referer provided by Indywidualny the screen is completely defect.
If I use your referer provided some postings ago without 'touch' in the options, FB works but looks like 1990.
If I use your referer with touch, FB just stays empty, the bar looks good and thats it, it just shows 'loading' all the time and freezes - as before the update of FaceSlim.

Any ideas? Since some weeks FB is completely unusable on my Q5 / looks like 1990 even in the browser.

**Comment by schmatzler** [comment]
@michaelsteffen1982 

Yes, FaceSlim is broken on an **actual** BlackBerry Q5. touch.facebook.com only works in a browser.

The useragent string from BlackBerry works for Android devices, though.

**Comment by michaelsteffen1982** [comment]
@natinusala It looks like the same user agent schmatzler provided some postings above - but it does not work for me - when I use this, as said, it just freezes and nothing is loaded. I see only the blue bar at the top (which looks fine) but thats it, nothing else is loading.

**Comment by schmatzler** [comment]
I was providing FaceSlim on BlackBerry World for a while and I know that it broke with version 3.1.
I know that for a fact, because version 3.1 was denied with "Bad appearance" by BlackBerry.

I suppose it is still broken now. BlackBerry's Android implementation is very buggy.

**Comment by michaelsteffen1982** [comment]
I am referring to your post in this thread here, 9 days ago:

> Change the user agent to:
>Mozilla/5.0 (BB10; Kbd) AppleWebKit/537.35+ (KHTML, like Gecko) Version/10.3.3.3057 Mobile >Safari/537.35+
>and it will work.
------

However, would be great if someone could fix this anyhow and give the users any solution...atm Facebook is completely unusable with a Q5. Or usable without messages...great.

**Comment by schmatzler** [comment]
Like I said, that user agent works on Android devices. On the Q5 you need to browse touch.facebook.com with the Blackberry browser.

I haven't been using my Q5 for a long time because of the bugs. I only dug it out to get FaceSlim to work properly on my Android phone. ;)

**Comment by michaelsteffen1982** [comment]
Still no solution for Q5? What I do not understand - why does touch.facebook.com in the browser still work - ui and messages - but not Faceslim? Depending on the referer it stucks on loading the page or messages cannot be opened anymore...
